htdocs/product/fournisseurs.php

```diff
@@ -959,7 +959,7 @@ function update_price_from_multicurrency() {
 					'pfp.barcode'=>array('label'=>$langs->trans("BarcodeValue"), 'enabled' => isModEnabled('barcode'), 'checked'=>0, 'position'=>16),
 					'pfp.packaging'=>array('label'=>$langs->trans("PackagingForThisProduct"), 'enabled' => getDolGlobalInt('PRODUCT_USE_SUPPLIER_PACKAGING'), 'checked'=>0, 'position'=>17),
 					'pfp.status'=>array('label'=>$langs->trans("Status"), 'enabled' => 1, 'checked'=>0, 'position'=>40),
-					'pfp.tms'=>array('label'=>$langs->trans("DateModification"), 'enabled' => isModEnabled('barcode'), 'checked'=>1, 'position'=>50),
+					'pfp.tms'=>array('label'=>$langs->trans("DateModification"), 'enabled' => '1', 'checked'=>1, 'position'=>50),
```

Comment:
@marc-dll 
The parameter enabled is a int and not a string.

Reply:
@lvessiller-opendsi Fixed, thank you !